actionview/lib/action_view/helpers/asset_tag_helper.rb

```diff
@@ -282,8 +285,11 @@ def image_alt(src)
       #   video_tag(["trailer.ogg", "trailer.flv"], size: "160x120")
       #   # => <video height="120" width="160"><source src="/videos/trailer.ogg" /><source src="/videos/trailer.flv" /></video>
       def video_tag(*sources)
```

Comment:
I'm not generally objecting to `extract_options!`, I agree that's out of scope. I'm objecting to using it in `video_tag` because the diff says it didn't use it before. But messing around in IRB showed me that we can't mix keyword arguments with a symbol keyed options hash without using `**options`, so that doesn't save us much here anyway. 👍 to keeping it like you had it 😁